x-pack/plugins/apm/public/components/app/transaction_details/WaterfallWithSummmary/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -159,6 +160,26 @@ function NameLabel({ item }: { item: IWaterfallSpanOrTransaction }) {
   }
 }
 
+function compressedSpanStyle(item: IWaterfallSpanOrTransaction, width: number, left: number): React.CSSProperties {
+  var itemBarStyle = { left: `${left}%`, width: `${width}%` };
```

Comment:
You can use pct which is available in https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/public/style/variables.ts#L27
```suggestion
  var itemBarStyle = { left: pct(left), width: pct(width} };
```